src/workbook/stakeholder-sheet/holdings-columns.ts

```diff
@@ -149,10 +149,32 @@ export class StockClassAsConvertedColumn {
       style: Styles.default,
     });
 
+    const conversion_value = `${
+      outstandingRange.getExtents().topLeftAddress
+    } * ${stockClass.conversion_ratio}`;
+
+    enum RoundingType {
+      CEILING = "CEILING",
+      FLOOR = "FLOOR",
+      NORMAL = "NORMAL",
+    }
+
+    let formula: string;
+    switch (stockClass.rounding_type) {
+      case RoundingType.CEILING:
+        formula = `CEILING(${conversion_value}, 1)`;
+        break;
+      case RoundingType.FLOOR:
+        formula = `FLOOR(${conversion_value}, 1)`;
+        break;
+      case RoundingType.NORMAL:
+      default:
+        formula = `ROUND(${conversion_value}, 0)`;
+        break;
+    }
```

Comment:
**Non-blocking:** This looks like a good candidate for some functional decomposition.